CONTRIBUTING.md

```diff
@@ -29,27 +124,31 @@ The documentation is translated into these languages:
 - Simplified Chinese (`zh-cn`)
 - Traditional Chinese (`zh-tw`)
 
-To find translations that need to be done, you can [filter for merged PRs](https://github.com/expressjs/expressjs.com/pulls?q=is%3Apr+is%3Aclosed+label%3Arequires-translation-es) that include the tag for your language, such as `requires-translation-es`.
+### Adding New Full Site Translations
+
+If you find a translation is missing from the list you can create a new one.
 
-When you contribute a translation, please reference the original PR. This helps the person merging your translation to remove the `requires-translation-es` tag from the original PR.
+To translate expressjs.com into a new language, follow these steps:
 
+1. Clone the [`expressjs.com`](https://github.com/expressjs/expressjs.com) repository.
+2. Create a directory for the language of your choice using its [ISO 639-1 code](http://www.loc.gov/standards/iso639-2/php/code_list.php) as its name.
+3. Copy `index.md`, `api.md`, `starter/`, `guide/`, `advanced/`, `resources/`, `4x/`, and `3x/`, to the language directory.
+4. Remove the link to 2.x docs from the "API Reference" menu.
+5. Update the `lang` variable in the copied markdown files.
+6. Update the `title` variable in the copied markdown files.
+7. Create the header, footer, notice, and announcement file for the language in the `_includes/` directory, in the respective directories, and make necessary edits to the contents.
+8. Create the announcement file for the language in the `_includes/` directory.
+9. Make sure to append `/{{ page.lang }}` to all the links within the site.
+10. Update the `CONTRIBUTING.md` and the `.github/workflows/translation.yml` files with the new language. 
```

Comment:
Adding the link so they know where to include it would be a good idea.
```suggestion
10. Update the [`CONTRIBUTING.md`](https://github.com/expressjs/expressjs.com/blob/gh-pages/CONTRIBUTING.md#contributing-translations) and the `.github/workflows/translation.yml` files with the new language. 
```